### DISCUSSION TITLE
是否支持markdown转换？

### DISCUSSION THREAD

**Discussion by sky-fly97** [lazyFrogLOL/llmdocparser#2] [Q&A]
我理解是gptpdf是整张图送进个gpt4o，只不过图片、table区域被打上了标签， llmdocparser是把每个区域都抠出来送进大模型？一个问题是推理成本会不会高很多，另一个问题是如果想要组装成markdown的话，哪种方式效果好呢？

**Comment by lazyFrogLOL** [comment]
> 我理解是gptpdf是整张图送进个gpt4o，只不过图片、table区域被打上了标签， llmdocparser是把每个区域都抠出来送进大模型？一个问题是推理成本会不会高很多，另一个问题是如果想要组装成markdown的话，哪种方式效果好呢？

第一个问题，推理成本我测过，差不多，比gptpdf略多一些。

第二个问题，组装markdown的话gptpdf更好。我这个项目是一步到位解决分块问题，单纯转换肯定用gptpdf。
